pdr_backend/analytics/get_predictions_info.py

```diff
@@ -98,7 +98,7 @@ def get_predictoors_info_main(
 
 @enforce_types
 def get_traction_info_main(ppss: PPSS, start_timestr: str, end_timestr: str):
-    table_name = NamedTable("pdr_predictions").fullname
+    table_name = Table("pdr_predictions").table_name
```

Comment:
I like this a lot. I'm wondering whether we should do this separately as a prep for this PR. Do you think it's possible? I would like to separate this renaming because it's much simpler to review and merge, it provides immediate benefits and results in a thinner, functionality-oriented, PR here.

Reply:
Hi @calina-c I have cherry-picked all of my changes and prepared a new PR that will make this easier to review [#1423](https://github.com/oceanprotocol/pdr-backend/pull/1423)